### PR TITLE
Populate client.ip for GitHub Enterprise audit logs

### LINKED ISSUE #157: GitHub audit logs can optionally include client IP addresses
Enterprise audit logs can now be configured to include client IP addresses in the dataset. We should map their `actor_ip` to `client.ip` in the managed log source.

https://docs.github.com/en/enterprise-cloud@latest/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/displaying-ip-addresses-in-the-audit-log-for-your-enterprise

### PR DESCRIPTION
**Pull request by timoguin**
This maps the optional `actor_ip` field to `client.ip`.

Closes #157.

Signed-off-by: Tim O'Guin <tim.oguin@cbsinteractive.com>

### CODE REVIEW

File: data/managed/log_sources/github_audit/log_source.yml [hunk 1]
```diff
@@ -141,6 +142,7 @@ transform: |2-
   }
 
   .client.geo.country_iso_code = del(.json.actor_location.country_code)
+  .client.ip = del(.json.actor_ip)
```
Comment (Samrose-Ahmed) [inline data/managed/log_sources/github_audit/log_source.yml:146]: we can also populate .related.ip

```suggestion

if .client.ip != null { 
      .related.ip = push(.related.ip, .client.ip) 
}
```

File: data/managed/log_sources/github_audit/log_source.yml [hunk 2]
```diff
@@ -2,6 +2,7 @@ name: github_audit
 schema:
   ecs_field_names:
     - client.geo.country_iso_code
+    - client.ip
     - ecs.version
```
Comment (Samrose-Ahmed) [inline data/managed/log_sources/github_audit/log_source.yml:6]: also add .related.ip to ecs fields

**Review by Samrose-Ahmed — COMMENTED** [review]
Excellent!

Small comment: we can also populate .related.ip when the ip exists. 

Feel free to make the change or let me know if you want me to add before merging.

File: data/managed/log_sources/github_audit/log_source.yml (hunk 2 above)
Comment (timoguin) [reply data/managed/log_sources/github_audit/log_source.yml:6]: done

File: data/managed/log_sources/github_audit/log_source.yml (hunk 1 above)
Comment (timoguin) [reply data/managed/log_sources/github_audit/log_source.yml:147]: also done

**Review by Samrose-Ahmed — APPROVED** [review]

**Pull request merged**